roles/tests-integration/tests/pool_integration.rs

```diff
@@ -94,3 +95,61 @@ async fn test_sniffer_interrupter() {
     assert_common_message!(&sniffer.next_message_from_downstream(), SetupConnection);
     assert_common_message!(&sniffer.next_message_from_upstream(), SetupConnectionError);
 }
+
+// covers
+// https://github.com/stratum-mining/stratum/blob/main/test/message-generator/test/translation-proxy/translation-proxy.json
+#[tokio::test]
+async fn translation_proxy() {
```

Comment:
I don't think it makes sense to add this to a file called `pool_integration.rs`

also, as we already discussed in the past with @Shourya742 , by having multiple tests in the same file, we end up executing them in parallel

this can cause problems with CPU miners

Reply:
most importantly, as we already discussed in the past with @Shourya742 , by having multiple tests in the same file, we end up executing all of them in parallel

this can cause problems with CPU miners